pkgs/development/python-modules/packageurl-python/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, fetchPypi, lib }:
+
+buildPythonPackage rec {
+  pname = "packageurl-python";
+  version = "0.9.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0mpvj8imsaqhrgfq1cxx16flc5201y78kqa7bh2i5zxsc29843mx";
+  };
+
+  pythonImportsCheck = [ "packageurl" ];
```

Comment:
```suggestion
  pythonImportsCheck = [ "packageurl" ];
  
  checkInputs = [ pytestCheckHook ];
```

Reply:
OK, added

Reply:
You shoyld also try whether the tests run successfully.

Reply:
Loving it that people are giving conflicting information. Very helpful :-/

Reply:
> Loving it that people are giving conflicting information. Very helpful :-/

To be a bit more constructive: the previous time I submitted it I was told to include what I included if I was not able to run tests. I am not able to run the tests at this point.

Reply:
Sorry, this was more about the missing argument `pytestCheckHook`.
If you post the output, we can help figure out how to make the tests run.

Reply:
I will need some help here. Note: I am not on NixOS, but Fedora. It has been 15 years since I did any serious work with Nix so my skills are very rusty.

Reply:
I get
```
Executing pytestCheckPhase
============================= test session starts ==============================
platform linux -- Python 3.9.2, pytest-6.1.2, py-1.9.0, pluggy-0.13.1
rootdir: /build/packageurl-python-0.9.4, configfile: setup.cfg
collected 258 items

tests/test_packageurl.py .....................................           [ 14%]
tests/contrib/test_get_path_segments.py ...                              [ 15%]
tests/contrib/test_purl2url.py ..                                        [ 16%]
tests/contrib/test_url2purl.py ......................................... [ 32%]
........................................................................ [ 60%]
........................................................................ [ 87%]
...............................                                          [100%]

============================= 258 passed in 0.84s ==============================
Finished executing pytestCheckPhase
```
So everything seems to work.